pnt-lut.c

```diff
@@ -23,6 +23,11 @@
 #define PGM_H 480
 #define PGM_SIZE ((PGM_W) * (PGM_H)) + 15
 
+#define TABLET_MAX_X 21600
+#define TABLET_MAX_Y 13500
+
+#define DEBUG 1 // printing x-y coordinates
```

Comment:
will need to disable if we don't want to fill our disk with raw coordinates

Reply:
conventional way of doing this is iirc is _not_ having this here, and instead

`CFLAGS+=-DDEBUG make`

just append to the `CFLAGS` that you want to define `DEBUG`, then can do `#ifdef DEBUG ... #endif` in source.

Reply:
fixed this, if you want debug now you'll have to edit the makefile / pass edited `CFLAGS`